src/lsp/utils.ts

```diff
@@ -0,0 +1,34 @@
+import * as roots from '../project-root';
+import * as vscode from 'vscode';
+
+export const findBestClojureProjectRoot = async (
```

Comment:
Can we either rename this to something more descriptive, or add a comment to explain what it does? It's not clear what "best" means.

Reply:
Ok, renamed and added a descriptive comment (bcefbe7a)